src/pages/Introduction/IntroductionView.js

```diff
@@ -0,0 +1,175 @@
+// @flow strict
+
+import React, { PureComponent } from 'react'
+import { t } from 'ttag'
+import classNames from 'classnames'
+import ViewSlider from 'react-view-slider'
+
+import {
+  JLink,
+  JRaisedButton,
+} from 'components/base'
+
+import {
+  setIntroductionValue,
+} from 'utils/introduction'
+
+import { Slide } from './components/Slide'
+
+import introductionViewStyle from './introductionView.m.scss'
+
+export type Props = {|
+|}
+
+type State = {|
+  activeId: number,
+|}
+
+const sliders = [
+  {
+    img: '/assets/feature/jwallet.png',
+    animationName: 'jwallet',
+    title: t`Jwallet`,
+    descr: [
+      t`A simple, fast and secure mobile wallet`,
```

Comment:
For proper translation it is better to have `<br>` inside and set the value with `dangerouslySetInnerHTML`